Makefile

```diff
@@ -64,7 +64,12 @@ requirements: ## install development environment requirements
 	pip install -qr requirements/pip-tools.txt
 	pip-sync requirements/dev.txt requirements/private.*
 
-test: clean ## run tests in the current virtualenv
+test: clean test-django test-scripts ## run tests in the current virtualenv
```

Comment:
In the django-app cookiecutter, it just uses pytest to test everything.  See https://github.com/edx/edx-cookiecutters/blob/master/cookiecutter-django-app/%7B%7Bcookiecutter.repo_name%7D%7D/Makefile#L60-L61  Can this use that pattern?

Reply:
I don't think so. I might be wrong, but I believe we need to run the Django tests in order to override the settings with the [`self.settings`](https://docs.djangoproject.com/en/dev/topics/testing/tools/#django.test.SimpleTestCase.settings) decorator.

Reply:
We use `pytest-django`.  Is that what usually makes this work?  See https://github.com/edx/edx-toggles/blob/master/requirements/test.in#L9. I don't know much about this config, but I would wonder how the cookie cutter works.  @jmbowman?

Reply:
Right! I understand now. I'm not too familiar with pytest or tox. I'll update the tox.ini to get the tests to pass with `pytest`.